Content.Server/GameObjects/Components/Projectiles/StunnableProjectileComponent.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Content.Server.GameObjects.Components.Mobs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Log;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.GameObjects.Components.Projectiles
+{
+    /// <summary>
+    /// Adds stun when it collides with an entity
+    /// </summary>
+    [RegisterComponent]
+    public sealed class StunnableProjectileComponent : Component, ICollideBehavior
+    {
+        public override string Name => "StunnableProjectile";
+
+        // See stunnable for what these do
+        private int _stunAmount;
+        private int _knockdownAmount;
+        private int _paralyzeAmount;
+
+        public override void ExposeData(ObjectSerializer serializer)
+        {
+            base.ExposeData(serializer);
+            serializer.DataField(ref _stunAmount, "stunAmount", 0);
+            serializer.DataField(ref _knockdownAmount, "knockdownAmount", 0);
+            serializer.DataField(ref _paralyzeAmount, "paralyzeAmount", 5);
```

Comment:
I think a non-0 default for just one stun type is going to be confusing and surprising. If I want to make a knockdown-only projectile, I have to specify that `paralyzeAmount: 0`, but not `stunAmount: 0`.

Reply:
Also unrelated to this review, but since paralyze is literally just stun and knockdown at the same time, you might want to expose slowdown instead, maybe